Plotting – again (#377)
tesseract-ocr/tesstrain: README.md

    unicharset       Create unicharset
    charfreq         Show character histogram
    lists            Create lists of lstmf filenames for training and eval

>     training         Start training (i.e. create .checkpoint files)

Reviewer: ```suggestion
    training         Start training
```

    unicharset       Create unicharset
    charfreq         Show character histogram
    lists            Create lists of lstmf filenames for training and eval

>     training         Start training (i.e. create .checkpoint files)

Reviewer: That additional information is an internal detail (which might change in the future). I suggest to keep the old text.

Author: The descriptions of `traineddata` and (new) `lstmeval` already refer to it, though.

    traineddata      Create best and fast .traineddata files from each .checkpoint file
    proto-model      Build the proto model
    tesseract-langdata  Download stock unicharsets

>     lstmeval         Evaluate .checkpoint models on eval dataset via lstmeval

Reviewer: I would prefer a different target name _evaluation_.

Add `MODEL_NAME` and `OUTPUT_DIR` and replace `data/foo` with the output directory if needed.

### Plotting CER

Training and Evaluation Character Error Rate (CER) can be plotted using Matplotlib:

    # Make OUTPUT_DIR/MODEL_FILE.plot_*.png
    make plot

All the variables defined above apply, but there is no explicit dependency on `training`.

> Still, the target hinges on the `LOG_FILE` intercepted during training (just will not trigger
> training itself). Besides the log file, this also evaluates the trained models (for each checkpoint)
> on the eval dataset. The latter is also available as an independent target `evaluation`:

Reviewer: ```suggestion
Still, the target will depends on the `LOG_FILE` captured during training (it will just not trigger training itself).
Besides the log file, the trained models (for each checkpoint) are also evaluated on the eval dataset.
The latter is also available as an independent target `evaluation`:
```

Author: `will depends`?

I don't like passive voice here, but pushed an improved paragraph.

    # Make OUTPUT_DIR/eval/MODEL_FILE*.*.log
    make evaluation

Plotting can even be done while training is still running, and  will depict the training status

> up to that point. (It can be rerun any time the `LOG_FILE` has changed or new checkpoints written.)

Author: I like `depict` better than `show` here, and `has changed` better than `is changed`.

> up to that point. (It can be rerun any time the `LOG_FILE` has changed or new checkpoints written.)

Reviewer: Deepl and I disagree.

> up to that point. (It can be rerun any time the `LOG_FILE` has changed or new checkpoints written.)

Author: Deepl does not know anything about tesstrain, and I am pretty confident in my assessment of what the English language allows me to express.

Besides, IMO it should not be your role as a maintainer to micromanage individual contributions to the letter. This is becoming extremely tedious – I thought this addition is long overdue and would be welcomed.
